README.md

```diff
@@ -1016,10 +1285,17 @@ const closeConnection = conn => Future((rej, res) => {
 ```
 
 #### finally
-##### `#finally :: Future a b ~> Future a c -> Future a b`
-##### `#lastly :: Future a b ~> Future a c -> Future a b`
-##### `.finally :: Future a c -> Future a b -> Future a b`
-##### `.lastly :: Future a c -> Future a b -> Future a b`
+
+<details><summary><code>finally :: Future a b ~> Future a c -> Future a b</code></summary>
+
+```hs
+finally :: Future a b ~> Future a c               -> Future a b
+lastly  :: Future a b ~> Future a c               -> Future a b
+finally ::               Future a c -> Future a b -> Future a b
+lastly  ::               Future a c -> Future a b -> Future a b
```

Comment:
The `#`/`.` are redundant, as the `~>` already indicates where a function lives. I think I should remove aliases from these though, as I believe they may add to the confusion.